pallets/gear-voucher/src/lib.rs

```diff
@@ -124,8 +127,8 @@ pub mod pallet {
     // Gas pallet error.
     #[pallet::error]
     pub enum Error<T> {
-        FailureToCreateVoucher,
-        FailureToRedeemVoucher,
+        InsufficientBalance,
+        InvalidVoucher,
```

Comment:
Too generic error. For example, we have `InvalidReservationId` which is overused in sys-calls

Reply:
Will be splitted into exact errors in following PRs as well